Dockerfile

```diff
@@ -93,5 +147,6 @@ RUN chown -R jovyan:jovyan /home/jovyan
 
 # Convert notebooks to the current format
 RUN find . -name '*.ipynb' -exec ipython nbconvert --to notebook {} --output {} \;
+RUN find . -name '*.ipynb' -exec ipython trust {} \;
```

Comment:
This should  make all notebooks trusted.

If you are fine with it, you can merge and kick the build.